tests/CMakeLists.txt

```diff
@@ -50,6 +52,15 @@ add_executable( es_test ${COMMON_SOURCES} ${ES_SOURCES} )
 target_link_libraries( es_test
                        graphene_chain graphene_app graphene_account_history graphene_elasticsearch
                        graphene_es_objects graphene_egenesis_none graphene_api_helper_indexes
+                       graphene_custom_operations
                        fc ${PLATFORM_SPECIFIC_LIBS} )
-
+                       
+file(GLOB CUSTOM_SOURCES "custom_operations/*.cpp")
+add_executable( custom_operations_test ${COMMON_SOURCES} ${CUSTOM_SOURCES} )
+target_link_libraries( custom_operations_test
```

Comment:
Please include the tests in the existing test programs. New ones are unlikely to be run regularly.

Reply:
moved them at https://github.com/bitshares/bitshares-core/pull/1926/commits/53fba25f4bd08b85c8e6295b332cd9e174e08a83